src/dev/impl/DevToys/DevToys.csproj

```diff
@@ -461,6 +469,12 @@
     <AppxManifest Include="Package.appxmanifest">
       <SubType>Designer</SubType>
     </AppxManifest>
+    <PRIResource Include="Strings\zh-Hans\ImageConverter.resw" />
```

Comment:
You can remove these lines, since line 478 includes everything (I know, have to change the file manually ... :( )